openmetadata-ui/src/main/resources/ui/src/components/Announcement/announcement.less

```diff
@@ -48,6 +48,8 @@
 
       .ant-avatar-group {
         z-index: 10;
+        margin-bottom: 5px;
+        margin-right: 5px;
```

Comment:
Can we use our size variable instead hardcoded CSS values?

Reply:
done